app/components/project-card.js

```diff
@@ -21,5 +21,7 @@ const {
 export default Component.extend({
   classNames: ['project-card'],
 
-  projectOrganizationMembers: Ember.computed.mapBy('project.organization.organizationMemberships', 'member')
+  projectCategories: Ember.computed.mapBy('project.projectCategories', 'category'),
+  projectOrganizationMembers: Ember.computed.mapBy('project.organization.organizationMemberships', 'member'),
+  projectSkills: Ember.computed.mapBy('project.projectSkills', 'skill')
```

Comment:
As an example, inside of a project's page may show its categories and skills. May use the same thing on the contributors page when approving a new contributor to see the overlap with their categories/skills and the projects'.